### PR TITLE
add discussion site for adc of measurement electronics

### PR DESCRIPTION
**Pull request by pascalguttmann**
PR to create the discussion for the ADC topics
==============================================

@ruchik-mahesh please add here the info of the adc parts you have shown today.

I will also put some info regarding the reference voltage topic in this PR (hopefully this weekend).

How to add info here
--------------------

1. `Pull` the newest state of the repository to your PC.
   In VS Code the `Sync` operation will also do that.
2. `Checkout` (also called `switch`) the branch of this PR: `feature/meas/discussion-site`
   a. You should see the file `./docs/workspace/meas/discuss/adc.md`
   b. When running `./go_mkdocs_serve.sh` the discussion site should be visible.
3. Add your content to `./docs/workspace/meas/discuss/adc.md`. (Feel free to change the file structure)
4. Commit your work (it will be committed to this branch)
5. Repeat 3. and 4. until satisfied
6. `Push` your changes to GitHub.
   In VS Code the `Sync` operation should also do that.
7. In the _Conversation_ section of this PR for each commit you made a new entry should be visible.
8. To get back to your other work, just `checkout`/`switch` to the branch you have been working on before.

You can repeat this procedure later to add more content. I suggest to do a `Pull` before you start making changes, so that the newest changes by others are already included in your working copy. (It avoids merge conflicts.)

### CODE REVIEW

File: docs/workspace/meas/discuss/adc.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+Parts Investigated
+------------------
+- ADS7038: [ADS7038 datasheet]
+- MAX1142: [MAX1142 datasheet]
+- LTC2308: [LTC2308 datasheet]
+- LTC2313-12: [LTC2313-12 datasheet]
+
+
```
Comment (ruchik-mahesh) [inline docs/workspace/meas/discuss/adc.md:25]: As discussed i have added all the info i have collected, you could give me some input so i can either go ahead with getting the price of these parts or look into more available ones.

File: docs/workspace/meas/discuss/adc.md [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+The performance of an ADC is primarily characterized by its bandwidth and signal-to-noise ratio (SNR). The bandwidth of an ADC is characterized primarily by its sampling rate.
+ADCs are chosen to match the bandwidth and required SNR of the signal to be digitized. If an ADC operates at a sampling rate greater than twice the bandwidth of the signal, then per the Nyquist–Shannon sampling theorem, near-perfect reconstruction is possible.
+ADCs are crucial in the modern world as they act as a bridge between the analog world and digital devices. They can be found as individual ADC ICs or embedded into a microcontroller.
+
 
 Performance Specification
 -------------------------
+•	Resolution -12/14 bit (or greater)
+•	Supply voltage: 5 V
+•	Sampling rate: 200 GHz (or greater)
```
Comment (pascalguttmann) [inline docs/workspace/meas/discuss/adc.md:14]: Typo: 200 kHz

**Review by pascalguttmann — COMMENTED** [review]
Thx for the addition of the parts :)

File: docs/workspace/meas/discuss/adc.md (hunk 1 above)
Comment (pascalguttmann) [reply docs/workspace/meas/discuss/adc.md:25]: Thanks :)

File: docs/workspace/meas/discuss/adc.md (hunk 2 above)
Comment (pascalguttmann) [reply docs/workspace/meas/discuss/adc.md]: fixed

**Comment by pascalguttmann** [comment]
@ruchik-mahesh I have made an investigation for ADC parts available on Mouser. The parts can measure input voltages from $U_{in} \in [0V, 5V]$, which I think would be beneficial to save another amplifier/attenuator from eg the current sensor.

@ruchik-mahesh Please review, and if you agree `approve` the PR.

**Review by ruchik-mahesh — DISMISSED** [review]
ADS8665:

Sampling Rate: Capable of up to 500 ksps, which is suitable for high-speed data acquisition requirements.
SPI Interface: Features an SPI interface with daisy-chain capability, allowing for the connection of multiple devices in series without the need for complex wiring or additional hardware.
Input Range: Offers software-selectable input ranges, both bipolar and unipolar, providing versatility in accommodating various signal types.
Overvoltage Protection: Includes input overvoltage protection up to ±20 V, enhancing the reliability and safety of the device under unexpected voltage conditions.
ADC121S021:

Sampling Rate: Offers a conversion speed ranging from 50 to 200 ksps, adequate for applications with moderate speed requirements.
Interface: Utilizes a high-speed serial interface, though it does not explicitly support SPI daisy-chaining.
Power Supply: Operates within a supply voltage range of 2.7 V to 5.25 V, catering to systems with standard power specifications.
Comparative Advantages of ADS8665:

The ADS8665’s higher sampling rate is advantageous for applications that demand rapid data collection.
Its SPI daisy-chain interface is beneficial for constructing complex DAQ systems with multiple ADCs, as it simplifies connectivity and reduces hardware complexity.
The programmable input ranges of the ADS8665 offer greater flexibility in signal compatibility, making it suitable for a broader range of applications.
The overvoltage protection feature of the ADS8665 ensures device integrity and longevity, even in the presence of voltage spikes.
I suggest we could go ahead with ADS8665, the interface you mentioned seemed to useful with collection and easier connection, i just made few points on both the device i compared, i looked into other parts as well but like you stated in this document i felt like comparing the two above and finally go with ADS8665 and on mouser the price for each part is around 3 euro

**Pull request merged**